src/App.jsx

```diff
@@ -1,12 +1,27 @@
-import { useState } from "react";
+import React, { useState } from "react";
+import Statistics from "./Statistics";
+import Button from "./Button";
+import StatisticLine from "./StatisticLine";
 
 const App = () => {
-  // save each button click in its own state
   const [good, setGood] = useState(0);
   const [neutral, setNeutral] = useState(0);
   const [bad, setBad] = useState(0);
 
-  return <div>code here</div>;
+  return (
+    <div>
+      <h1>Give Feedback</h1>
+      <Button onClick={() => setGood(good + 1)} text="Good" />
+      <Button onClick={() => setNeutral(neutral + 1)} text="Neutral" />
+      <Button onClick={() => setBad(bad + 1)} text="Bad" />
+
+      {good + neutral + bad > 0 ? (
```

Comment:
Define a function or store this calculation in a constant instead of directly using this statement (`good + neutral + bad > 0`) here.